apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.ts

```diff
@@ -37,10 +43,12 @@ export class DateRangePickerComponent {
         status,
         this.pickerFormControl.errors,
       );
+      this.#changeDetector.detectChanges();
```

Comment:
`markForCheck` was not updating the playground to show the changes on the controls when the validators changed without this. I wasn't able to find anything I could do in the playground or datepicker that would pick up that change without the hard `detectChanges` here. Since this was the playground - I didn't worry to much about it. Happy to chat if you'd like around this one.